src/libs/Middleware/SaveResponseInOnyx.js

```diff
@@ -14,50 +14,52 @@ const requestsToIgnoreLastUpdateID = ['OpenApp', 'ReconnectApp', 'GetMissingOnyx
  * @returns {Promise}
  */
 function SaveResponseInOnyx(requestResponse, request) {
-    return requestResponse.then((response) => {
-        // Make sure we have response data (i.e. response isn't a promise being passed down to us by a failed retry request and response undefined)
-        if (!response) {
-            return;
-        }
-        const onyxUpdates = response.onyxData;
-
-        // Sometimes we call requests that are successfull but they don't have any response or any success/failure data to set. Let's return early since
-        // we don't need to store anything here.
-        if (!onyxUpdates && !request.successData && !request.failureData) {
-            return Promise.resolve(response);
-        }
-
-        // If there is an OnyxUpdate for using memory only keys, enable them
-        _.find(onyxUpdates, ({key, value}) => {
-            if (key !== ONYXKEYS.IS_USING_MEMORY_ONLY_KEYS || !value) {
-                return false;
+    return requestResponse
```

Comment:
@tienifr Why are having this much diff here? I know we are only adding catch to this? 

Can we remove the diff? Any specific reason to have?

Reply:
Thank you. I also verified. No changes.